spec/core/v2/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -784,14 +825,49 @@ function writeAcknowledgement(
     // Note that the event should be emitted by this function only in the asynchrounous ack case. Otherwise the event is emitted during the onReceive 
     packet=getPacket(destChannelId,sequence)
     if(packet!=nil){
-        emitLogEntry("writeAcknowledgement", {
-        sequence: packet.sequence,
+        // construct acknowlegement event by concatenating each app acknowledgment together with a "/" delimiter
```

Comment:
Since we emit separate events for multiple payloads, could we not do the same here? Is it because it would be simpler for an event handler to always have to deal with a single writeAcknowledgement event per packet?

Reply:
We could do this yes. Think I just did this before thinking about the payload case